keyboards.py

```
@@ -0,0 +1,139 @@
+from aiogram.types import InlineKeyboardMarkup, InlineKeyboardButton
+from config import TARIFF_PLANS, PLATFORMS, DISTRIBUTIONS
```

Ruff (0.12.2)

`config.DISTRIBUTIONS` imported but unused

Remove unused import: `config.DISTRIBUTIONS`

(F401)